certified-connectors/Cornerstone Learning vILT/apiProperties.json

```diff
@@ -0,0 +1,167 @@
+{
+	"properties": {
```

Comment:
Kindly correct the apiProperties.json file as per the schema here https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json.

Please refer this validation error message - ##[error]@{level=Error; message=JSON is valid against no schemas from 'oneOf'.; valuePath=line: 5 position: 11 path: '[0].properties.connectionParameters.url'; value=}

Reply:
Corrected in last commit. Thanks!